apps/web/vibes/soul/components/product-chip/index.tsx

```diff
@@ -0,0 +1,64 @@
+'use client'
+
+import Image from 'next/image'
+import { useEffect, useState } from 'react'
+
+import clsx from 'clsx'
+
+import Icon from '@/vibes/soul/components/icon'
+
+type ProductCard = {
+  name: string
+  image: string
+}
+
+type Props = {
+  product: ProductCard
+  onDismiss?: () => void
+}
+
+const ProductChip = function ProductChip({ product, onDismiss }: Props) {
```

Comment:
If this component is only being used in the `CompareBar` component?

Reply:
Offline synced with @apledger and we want to remove instances like this, when a component is being used only in one other component. Let's move this to CompareBar.